Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,21 @@
+const isEmpty = require("./isEmpty.js");
+
+function isFloat(n) {
+  return !isNaN(n) && n % 2 !== 0;
+}
+
 function sum(elements) {
+  let sum = 0;
+  if (isEmpty(elements)) {
+    return 0;
+  }
+  for (let num of elements) {
+    isFloat(num)
+      ? (sum += parseFloat(num.toFixed(2)))
```

Comment:
I wanted to get two decimal places to make the result of the calculation correct but because this ```num.toFixed(2)``` is a string that's why I am converting it back to a float. I do not know whether this is the best way of doing it or not.  How can I make it better?